src/components/Api.js

```diff
@@ -0,0 +1,75 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  _handleServerResponse(res) {
+    return res.ok ? res.json() : Promise.reject(`Error: ${res.status}`);
+  }
+
+  _request(url, options) {
+    return fetch(url, options).then(this._handleServerResponse);
+  }
+
+  getInitialCards() {
+    const Cardinfo = this._request(`${this._baseUrl}/cards`, {
```

Comment:
`Cardinfo`

Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)